Source/CarthageKit/Project.swift

```diff
@@ -117,6 +118,10 @@ public enum ProjectEvent {
 	/// Building the project is being skipped, since the project is not sharing
 	/// any framework schemes.
 	case skippedBuilding(ProjectIdentifier, String)
+
+	/// Installing of a binary framework is being skipped because of an inability
+	/// to verify that it was built with a compatible Swift version.
+	case skippedInstallingBinaries(ProjectIdentifier, String)
```

Comment:
How about `skippedIncompatibleBinary`?

Also what's the `String` here? Can you add a label to compensate for the poor type information?